src/GrantType/GrantTypeInterface.php

```diff
@@ -12,4 +12,16 @@
      * @return AccessToken
      */
     public function getToken();
+
```

Comment:
I have filling this two config methods not belong to GrantType, what do you think?